game/theater/base.py

```diff
@@ -29,6 +31,10 @@ def __init__(self):
         self.commision_points: Dict[Type, float] = {}
         self.strength = 1
 
+    @property
+    def total_shorad_cost(self) -> int:
```

Comment:
This is a count, not a cost.

Reply:
Property not needed anymore, will be removed.